src/plugins/unified_histogram/public/chart/chart.tsx

```diff
@@ -65,20 +77,28 @@ export function Chart({
   className,
   services,
   dataView,
-  lastReloadRequestTime,
+  query: originalQuery,
+  filters: originalFilters,
+  timeRange: originalTimeRange,
   request,
   hits,
   chart,
   breakdown,
   appendHitsCounter,
   appendHistogram,
+  disableAutoFetching,
+  disableTriggers,
+  disabledActions,
```

Comment:
to understand it correctly, we offer `disableTriggers` + `disableActions` just for alignment, but don't set custom values ... I'm interested, what's the user case to set them?

Reply:
Yeah this change was more in anticipation of future needs from other consumers. The main use case would be to disable adding/removing filters from the legend, disable time range scrubbing, etc. for cases where filters or the time range might not be modifiable. These actions also integrate with the search service by default which might not be used in some situations.